docs/madr/decisions/074-naming-internal-envoy-resources.md

```diff
@@ -0,0 +1,152 @@
+# Standardized Naming for internal xDS Resources
+
+* Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/13266
+
+## Context and Problem Statement
+
+We use resource names to identify listeners, clusters, routes, virtual hosts and secrets in the xDS APIs.
+These names appear in metrics, logs, and debug endpoints (config_dump), and they are often used for monitoring, filtering, and diagnostics.
+
+It's important to know the difference between internal resources (the ones that are defined by Kuma itself, and they usually do not concern the end users) and external resources (the ones that are defined indirectly by the user and the users are interested in).
```

Comment:
nit: calling them "external" doesn't feel right, maybe "internal" and "user-defined"?